gazebo_plugins/include/gazebo_plugins/gazebo_ros_utils.h

```diff
@@ -41,6 +41,14 @@
 #include <gazebo/sensors/Sensor.hh>
 #include <ros/ros.h>
 
+#ifndef GAZEBO_SENSORS_USING_DYNAMIC_POINTER_CAST
+# if GAZEBO_MAJOR_VERSION >= 7
+#define GAZEBO_SENSORS_USING_DYNAMIC_POINTER_CAST using std::dynamic_pointer_cast
```

Comment:
just for my own education, what's the reasoning behind using `using` as opposed to making `xxx::dynamic_pointer_cast` a macro?